### PR TITLE
Add Receipt Wrangler

### PR DESCRIPTION
**Pull request by Noah231515**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/receipt-wrangler.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: "Receipt Wrangler"
+website_url: "https://receiptwrangler.io"
+source_code_url: "https://github.com/Receipt-Wrangler"
+description: "Open Source, Self hosted, Receipt Management."
```
Comment (kokomo123) [inline software/receipt-wrangler.yml:4]:
```suggestion
description: " An easy to use Receipt manager. Allowing users to create receipts effortlessly and quickly, categorize and more."
```
This should be a better description, feel free to correct others 👍🏻

Comment (Noah231515) [reply software/receipt-wrangler.yml]: 👍👍 Thanks for the suggestion. Updated!

**Comment by kokomo123** [comment]
1 question before I approve this, going over the repos, is the receipt manager powered by an external AI, like ChatGPT, or locally made AI? Just to make sure we can either leave in or omit the depends-3rd-party: true flag

**Comment by kokomo123** [comment]
Solved my question for myself, yep! I shall approve this PR as there's no further issues from me.

**Review by kokomo123 — APPROVED** [review]

File: software/receipt-wrangler.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: "Receipt Wrangler"
+website_url: "https://receiptwrangler.io"
+source_code_url: "https://github.com/Receipt-Wrangler"
+description: "An easy to use Receipt manager, powered by AI. Allows users to create receipts effortlesly and quickly, catagorize and more."
```
Comment (nodiscc) [inline software/receipt-wrangler.yml:4]:
```suggestion
description: "Easy-to-use receipt manager, powered by AI. Allows users to create receipts effortlessly and quickly, categorize and more."
```

Minor syntax/typo fixes

File: software/receipt-wrangler.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+licenses:
+  - AGPL-3.0
+platforms:
+  - Go
+  - Python
+  - Docker
```
Comment (nodiscc) [inline software/receipt-wrangler.yml:10]:
```suggestion
platforms:
  - Docker
```

Docker is the only supported installation method https://github.com/Receipt-Wrangler#step-1-set-up-docker-composeyaml

**Review by nodiscc — APPROVED** [review]

**Pull request merged**